java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -791,11 +791,17 @@ public void Lambdas() throws Exception {
     JavaSymbol parenthMethod = result.symbol("parenth");
     ParenthesizedTree parenthesizedTree = (ParenthesizedTree) ((ReturnStatementTree) ((MethodTree) parenthMethod.declaration()).block().body().get(0)).expression();
     assertThat(parenthesizedTree.symbolType()).isSameAs(Ftype);
+    assertThat(result.symbol("s", 33).type().is("java.lang.String"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Complete the assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2970)